### PR TITLE
Adding Maalselva metadata

### PR DESCRIPTION
**Pull request by ZofiaRudjord**

### CODE REVIEW

File: src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py [hunk 1]
```diff
@@ -339,6 +344,67 @@ def loggers_variables() -> List[VariablesCreate]:
...
+            "of a solution. pH is formally a measure of the activity "
+            "of dissolved hydrogen ions (H+). Solutions in which the "
+            "concentration of H+ exceeds that of OH- have a pH value "
+            "lower than 7.0 and are known as acids.",
+            "variablecode": "PhValue",
+            "nodatavalue": -9999,
+        },
+        {
+            "variabletypecv": "Chemistry",
+            "variablenamecv": "Carbon dioxide",
```
Comment (knl88) [inline src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:368]: How is this connected to carbon dioxide? initially I would have guessed this is digitally sampled chromophoric dissolved organic matter, but haven't checked

File: src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py [hunk 2]
```diff
@@ -339,6 +344,67 @@ def loggers_variables() -> List[VariablesCreate]:
...
+            "variabledefinition": "Carbon dioxide",
+            "variablecode": "CDOMdigitalFinal",
+            "nodatavalue": -9999,
+        },
+        {
+            "variabletypecv": "Chemistry",
+            "variablenamecv": "Carbon dioxide",
+            "variabledefinition": "Carbon dioxide",
+            "variablecode": "CDOManalogFinal",
```
Comment (knl88) [inline src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:377]: same as above?

Comment (knl88) [reply src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:377]: analog sample

File: src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py (hunk 1 above)
Comment (ZofiaRudjord) [reply src/odm2_postgres_api/metadata_init/data/loggers/loggers_metadata.py:368]: Good catch, I think this is a mistake. CDOM is dissolved organic matter

**Pull request merged**